integration/tests/epochs/recover_epoch/suite.go

```diff
@@ -0,0 +1,24 @@
+package recover_epoch
+
+import (
+	"github.com/onflow/flow-go/integration/tests/epochs"
+)
+
+// Suite encapsulates common functionality for epoch integration tests.
+type Suite struct {
+	epochs.BaseSuite
+}
+
+func (s *Suite) SetupTest() {
+	// use a longer staking auction length to accommodate staking operations for joining/leaving nodes
+	// NOTE: this value is set fairly aggressively to ensure shorter test times.
+	// If flakiness due to failure to complete staking operations in time is observed,
+	// try increasing (by 10-20 views).
+	s.StakingAuctionLen = 2
+	s.DKGPhaseLen = 50
+	s.EpochLen = 250
```

Comment:
Just a note: When we get to making this integration test case, we'll likely be able to set these more aggressively (shorter). We can discuss later